e2e-tests/benchmarking/benchmark.txt

```diff
@@ -0,0 +1,149 @@
+
```

Comment:
Why should this file be checked in?

Reply:
It is not necessary, it can be deleted after the scrip is finished.